list-sensors.cpp

```diff
@@ -216,10 +218,17 @@ int main(int argc, char* argv[])
     }
 #endif
 
+    std::string root_path = SENSORS_PATH;
+    if (optind < argc)
+    {
+        root_path += "/";
+        root_path += argv[optind];
```

Comment:
Don't we want to filter out extra slashes from the argument? I believe we only need to accept  `[A-Za-z0-9_]` here.

Reply:
FIxed